### ISSUE TITLE
(Win 10) Clicking on Text Opens File Explorer

### ISSUE THREAD

**Issue by tfevens** [beyondcode/invoker-community#73]
When clicking on a `string`, `date`, `datetime` field in the main Model table, quick view (right hand panel) or individual Model view table.

![Invoker-date](https://user-images.githubusercontent.com/701763/106795347-57a7a500-6630-11eb-9ea5-b05672b5c967.gif)
------
**OS**: win32 (Win 10)
**Invoker Version**: 2.0.2
**Laravel Version**: 8.25.0

Gif shows clicking only on date/datetime, but string has been confirmed as well.

**Comment by mpociot** [comment]
This is so weird 😂 
I'll look into this

**Comment by rumspeed** [comment]
I am experiencing the same issue.

**Comment by nickturrietta** [comment]
Same issue here on Windows 10:
https://www.youtube.com/watch?v=WZET0vifLu0

**Comment by nickturrietta** [comment]
I am having the same issue.
![bug](https://user-images.githubusercontent.com/6426374/106814628-8bca9800-6627-11eb-960e-bfc67d373ce3.gif)

**Comment by mpociot** [comment]
Is this a duplicate of #66?
In #66, it's about a double click, while here it says "click".

**Comment by mpociot** [comment]
Nevermind, I just fixed the issue (the update will be released tomorrow, with a bunch of other things that pop up in between).

The click event is in place to allow clicking on URLs, like these:

![image](https://user-images.githubusercontent.com/804684/106817189-cf002780-6676-11eb-8806-dd6728670ee2.png)

I noticed that the click handler was always attached - even if the URL is invalid. It seems like that's not a problem on macOS/linux, but Windows does this.

Thanks for reporting this!
